guides/common/assembly_common-deployment-scenarios.adoc

```diff
@@ -0,0 +1,13 @@
+include::modules/con_common-deployment-scenarios.adoc[]
+
+include::modules/con_single-location-with-segregated-subnets.adoc[leveloffset=+1]
+
+include::modules/con_multiple-locations.adoc[leveloffset=+1]
+
+include::modules/con_content-view-scenarios.adoc[leveloffset=+1]
+
+ifdef::satellite[]
```

Comment:
```suggestion
ifdef::satellite[]
```

This should also be possible with Foreman/Katello and orcharhino.

Reply:
So, `ifdef::katello,orcharhino,satellite[]`? That's what I did, let me know if I misunderstood.